ceramic-config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "anchor": {
+    "anchor-service-url": "https://cas-qa.3boxlabs.com",
+    "ethereum-rpc-url": "https://goerli.infura.io/v3/b6685df41e1647c4be0046dfa62a020b"
+  },
+  "http-api": {
+    "cors-allowed-origins": [".*"]
+  },
+  "ipfs": {
+    "mode": "bundled"
+  },
+  "logger": {
+    "log-level": 0,
+    "log-to-files": true
+  },
+  "metrics": {
+    "metrics-exporter-enabled": false,
+    "metrics-port": 9090
+  },
+  "network": {
+    "name": "dev-unstable"
+  },
+  "node": {},
+  "state-store": {
+    "mode": "fs",
+    "local-directory": "/Users/stephaniehuynh/.ceramic/statestore/"
+  },
+  "indexing": {
+    "db": "sqlite:///Users/stephaniehuynh/.ceramic/indexing.sqlite",
+    "allow-queries-before-historical-sync": true,
+    "models": [
```

Comment:
We're going to be removing the ability to specify the models to index via the config file soon.  Can you update this to use the admin api to specify these models instead?